spec/poltergeist_helper.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
```

Comment:
You're right about the scope. I don't think that the smoke tests invalidate any of those specs, though.